src/js/views/createCertificate/layout/style.js

```diff
@@ -18,4 +23,9 @@ export default makeStyles(theme => ({
       minWidth: '16rem',
     },
   },
+  inputCommand: {
+    fontSize: 14,
+    marginLeft: theme.spacing(1),
```

Comment:
I think that a `marginRight` would be great and maybe you can set a fixed width for the input.